staff/marc-uson/youtube-channel-template/index.html

```diff
@@ -0,0 +1,241 @@
+<!DOCTYPE html>
```

Comment:
@Markuson I'm not sure if these files should be here, it appears to me that you had a wrong folder while switching between branches and this "te la ha liao parda" :/